MacPass/MPAutotypeDaemon.m

```diff
@@ -241,6 +242,10 @@ - (void)_performAutotypeForEntry:(KPKEntry *)entryOrNil {
     [NSUserNotificationCenter.defaultUserNotificationCenter deliverNotification:notification];
     self.userActionRequested = NSDate.date.timeIntervalSinceReferenceDate;
     [NSNotificationCenter.defaultCenter addObserver:self selector:@selector(_didUnlockDatabase:) name:MPDocumentDidUnlockDatabaseNotification object:nil];
+    if (!NSApp.isActive) {
```

Comment:
The idea of just opening the last document is nice, but might be undesired behaviour but this could be a preference for autotype.

Reply:
I think this idea is very good.